apps/src/aiComponentLibrary/chatMessage/ChatMessage.tsx

```diff
@@ -40,31 +38,12 @@ const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({
   const hasWarningStyle = status === Status.PII_VIOLATION;
 
   const getDisplayText: string = useMemo(() => {
```

Comment:
Great refactoring! As discussed offline, we may want to revisit this after the break and refactor a bit more (e.g. moving the footer content to its own prop rather than passing as children) since this is now being actively used by AI Chat and TA

Reply:
Noted in the follow-up work list!